bool finish
---
float64 rl_neckangle